ios/RNYotiDocScanManager.m

```diff
@@ -57,6 +63,14 @@ - (NSString * _Nonnull)sessionTokenFor:(YotiSDKNavigationController * _Nonnull)n
     return self.sessionToken;
 }
 
+- (ServerLocation)serverLocationFor:(YotiSDKNavigationController * _Nonnull)navigationController {
```

Comment:
Hi @asklausen !
As we were talking about this morning, I'm a little bit sticky with this config...could you help me with it pls? I'm not sure how I have to manage an enum defined in Swift from Obj-C 😅  And I'm pretty sure you're going to smash it in 1 min :happy:
Thanks!